documentation/source/users/rmg/surfaces.rst

```diff
@@ -25,13 +25,13 @@ This block should also contain the reference adatom binding energies
 Here is an example catalyst properties block for Pt(111)::
 
     catalystProperties(
-        bindingEnergies = { 
-                            'H': (-2.479, 'eV/molecule'),
-                            'O': (-3.586, 'eV/molecule'),
-                            'C': (-6.750, 'eV/molecule'),
-                            'N': (-4.352, 'eV/molecule'),
+        bindingEnergies = {
```

Comment:
Why are these changing?

Reply:
Does the commit message to https://github.com/ReactionMechanismGenerator/RMG-Py/pull/2017/commits/3e19e56ce5d9b3c97202deffd04451cc3cf2aeaf explain it sufficiently?

Reply:
The commit hashes keep changing as things get rebased to keep up with master, so I'll copy and paste it here:
> **Correct binding energies in surface documentation.**
> PR #1996 updated the binding energies and surface site density
in the example input files, which we had initially overlooked.
This now updates the documentation, which had also been missed.
>
>I also added a reminder note in the source code so we are less
likely to forget again.

So the numbers in the documentation are changing to bring them in line with the source code and example files.
Is this now good to go?

Reply:
looks good to me!